src/components/Carousel/Carousel.stories.tsx

```diff
@@ -375,3 +383,53 @@ Scroller_Custom_Buttons.args = {
   style: { background: 'transparent' },
   type: 'scroll',
 };
+
+Scroller_With_Tooltips.args = {
+  ...carouselArgs,
+  carouselScrollMenuProps: {
+    children: sampleList.map((item: SampleItem) => (
+      <Card bordered height={344} key={item.key} tabIndex={0} width={280}>
+        <div
+          style={{
+            alignItems: 'center',
+            display: 'flex',
+            flexDirection: 'column',
+            gap: 24,
+            height: '100%',
+            justifyContent: 'center',
+            width: '100%',
+          }}
+        >
+          {item.name}
+          <Tooltip
+            content={TOOLTIP_INFO_TEXT}
+            portal
+            size={TooltipSize.Medium}
+            theme={TooltipTheme.dark}
+            tooltipStyle={{ fontSize: 14 }}
+            wrapperStyle={{ width: 'fit-content' }}
+          >
+            <div
+              style={{
+                display: '-webkit-box',
+                overflow: 'hidden',
+                textAlign: 'center',
+                textOverflow: 'ellipsis',
+                WebkitBoxOrient: 'vertical',
+                WebkitLineClamp: 4,
+                whiteSpace: 'pre-line',
+                width: '100%',
+              }}
+            >
+              {TOOLTIP_INFO_TEXT}
+            </div>
```

Comment:
Replace with useTruncate hook implementation. Would be a good demo of the hook only.